source/python/neuropod/tests/test_tensorflow_v2_packaging.py

```diff
@@ -0,0 +1,115 @@
+# Copyright (c) 2021 UATC, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import tensorflow as tf
+import unittest
+from testpath.tempdir import TemporaryDirectory
+
+from neuropod.packagers import create_tensorflow_neuropod
+from neuropod.tests.utils import get_addition_model_spec, check_addition_model
+from neuropod.utils.eval_utils import RUN_NATIVE_TESTS
+
+
+def create_tf_addition_model():
+    """
+    A simple addition model
+    """
+
+    class Adder(tf.Module):
+        @tf.function(
+            input_signature=[
+                tf.TensorSpec(shape=None, dtype=tf.float32),
+                tf.TensorSpec(shape=None, dtype=tf.float32),
+            ]
+        )
+        def add(self, x, y):
+            return {"out": x + y}
+
+    return Adder()
+
+
+@unittest.skipIf(tf.__version__[0] != "2", "Skipping TF 2.x tests for TF 1.x")
```

Comment:
Why skip tests for TF 1.x? In the TF saved_model code, you made changes to specifically support multiple versions.

Reply:
AFAIK, the way to create a TF saved model from Python is different in TF 1.x vs 2.x.

The pressing usecase is for TF 2.x so maybe we can just create an issue to add savdmodel tests for TF 1.x?

Reply:
That sounds good to me

Reply:
Created https://github.com/uber/neuropod/issues/496